src/XRay/XRayJSON.ts

```diff
@@ -193,9 +178,6 @@ namespace XRayJSON {
             (
                 (json as Breakdowns).assetAllocation.length === 0 ||
                 isAssetAllocation((json as Breakdowns).assetAllocation[0])
-            ) || (
-                !(json as Breakdowns).regionalExposure ||
-                checkRegionalExposure((json as Breakdowns).regionalExposure)
             ) || (
```

Comment:
That goes too far with optimization. What I meant is that this can be replaced with 
```
!(json as Breakdowns).regionalExposure ||
isRegionalExposure((json as Breakdowns).regionalExposure?.[0])
```